niceml/config/hydra.py

```diff
@@ -73,17 +73,21 @@ def __init__(
         target_class,
         description: Optional[str] = None,
         default_value: Optional[dict] = None,
+        example_value: Optional[dict] = None,
```

Comment:
This is a very common name. What is it good for? A comment can be helpful.

Reply:
This example value should be used for documentation